Buddies/Activities/ActivityTableVC.swift

```diff
@@ -173,5 +173,15 @@ class ActivityTableVC: UITableViewController {
     // "abstract"
     // Must call loadData() once displayIds is set
     func fetchAndLoadActivities(){}
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        let selectedIndex = self.tableView.indexPath(for: sender as! UITableViewCell)
+        if let nav = segue.destination as? UINavigationController,
```

Comment:
Why do we use a nav controller when we present modally anyway?

Reply:
Because the nav controller is required for the header. I spent about 2 hours trying to do a header without the UINavigationController, and there were lots of different issues. For instance, a manual nav header bar doesn't go under the status bar among other things. The nav controller does it by default.

Reply:
Sweet, makes sense